corehq/ex-submodules/soil/__init__.py

```diff
@@ -152,20 +154,46 @@ def set_progress(cls, task, current, total):
         except IntegrityError:
             # Not called in task context just pass
             pass
-    
+
     @classmethod
     def create(cls, payload, **kwargs):
         """
-        Create a Download object from a payload, plus any additional arguments 
+        Create a Download object from a payload, plus any additional arguments
         to pass through to the constructor.
         """
         raise NotImplementedError("This should be overridden by subclasses!")
 
 
+class MultipleTaskDownload(DownloadBase):
```

Comment:
This might be the wrong name for this, since I doubt there will ever be files to download once these tasks complete...